bin/graphics-core22-cleanup

```diff
@@ -3,6 +3,32 @@ set -euo pipefail
 
 SELF="$( cd -- "$(dirname "$0")/.." ; pwd -P )"
 
+TMP_FOLDER=/tmp/snap-lists
```

Comment:
You probably want something like `mktemp -d` here.

Reply:
Good point.

Reply:
Done.